.github/workflows/wfc_backport.yml

```diff
@@ -83,11 +83,12 @@ jobs:
           git config user.email "${GH_EMAIL}"
       - name: cherry-pick
         run: |
-          echo LABELS=${{ matrix.branch }} >> $GITHUB_ENV
+          INHERITED_LABELS=$(gh pr view --repo ${{ github.repository }} ${{ env.PR_NUMBER }} --json 'labels' --jq '[ .labels[] | select (.name != "backport") | .name ] | join(",")')
+          echo LABELS=$INHERITED_LABELS,${{ matrix.branch }} >> $GITHUB_ENV
```

Comment:
is it possible for `INHERITED_LABELS` to be empty? if so then we probably append label to `LABELS` using join

Reply:
it can be empty, then it will be 

```
LABELS=,release-2.6
```

and I think that worked fine. Do you want me to if/else that just to be 100% sure?

Reply:
I guess we'll notice this pretty quickly on the backport if labels are invalid. So I'd just merge it